### ISSUE TITLE
Improve editing page accessibility

### ISSUE THREAD

**Issue by ChiuchiuSorin** [xwikisas/application-collabora#101]
Currently, there are several accessibility issues on the Collabora edit page:

1. Missing landmarks for content
    - Landmark roles provide programmatic access to sections of a web page, making it easy for the user to understand the structure of the page and easily navigate to different sections: https://www.w3.org/WAI/ARIA/apg/practices/landmark-regions/

2. XWiki logo misses accessible name.

3. iframe misses title attribute    - An inline frame’s title provides a label for the frame that describes its content to the user. With this information a user can determine which frame to enter and explore in detail: https://www.w3.org/WAI/WCAG22/Techniques/html/H64
